coursier: add new documentation pages (#9728)
tldr-pages/tldr: pages/common/cs-resolve.md

# cs resolve

> Resolve lists the transitive dependencies of one or more other dependencies.
> More information: <https://get-coursier.io/docs/cli-resolve>.

- Resolve lists of transitive dependencies of two dependencies:

`cs resolve {{group_id1}}:{{artifact_id1}}:{{artifact_version1}} {{group_id2}}:{{artifact_id2}}:{{artifact_version2}}`

- Resolve lists of transitive dependencies of a package by the dependency tree:

`cs resolve --tree {{group_id}}:{{artifact_id}}:{{artifact_version}}`

> - Resolve dependency tree in a reverse order (from a dependency to its dependencies):
> 
> `cs resolve --reverse-tree {{group-id}}:{{artifact_id}}:{{artifact_version}}`

Reviewer: ```suggestion
- Resolve dependency tree in a reverse order (from a dependency to its dependencies):

`cs resolve --reverse-tree {{group_id}}:{{artifact_id}}:{{artifact_version}}`
```

Mandatory fix: [broken placeholder](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#naming): snake case expected.

# cs resolve

> Resolve lists the transitive dependencies of one or more other dependencies.
> More information: <https://get-coursier.io/docs/cli-resolve>.

> - Resolve lists of transitive dependencies of two dependencies:
> 
> `cs resolve {{groupId1}}:{{artifactId1}}:{{artifactVersion1}} {{groupId2}}:{{artifactId2}}:{{artifactVersion2}}`

Reviewer: ```suggestion
- Resolve lists of transitive dependencies of two dependencies:

`cs resolve {{group_id1}}:{{artifact_id1}}:{{artifact_version1}} {{group_id2}}:{{artifact_id2}}:{{artifact_version2}}`
```

Mandatory fix: [broken placeholder](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#naming): snake case expected.

> - Resolve lists of transitive dependencies of a package by the dependency tree:
> 
> `cs resolve --tree {{groupId}}:{{artifactId}}:{{artifactVersion}}`

Reviewer: ```suggestion
- Resolve lists of transitive dependencies of a package by the dependency tree:

`cs resolve --tree {{group_id}}:{{artifact_id}}:{{artifact_version}}`
```

The same issue as above.

> - Resolve dependency tree in a reverse order (from a dependency to its dependencies):
> 
> `cs resolve --reverse-tree {{groupId}}:{{artifactId}}:{{artifactVersion}}`

Reviewer: ```suggestion
- Resolve dependency tree in a reverse order (from a dependency to its dependencies):

`cs resolve --reverse-tree {{group-id}}:{{artifact_id}}:{{artifact_version}}`
```

The same issue as above.

> - Print all the libraries that depends on a particoular library:
> 
> `cs resolve {{groupId}}:{{artifactId}}:{{artifactVersion}} --what-depends-on {{searchedGroupId}}:{{searchedArtifactId}}`
> 

Reviewer: ```suggestion
- Print all the libraries that depends on a specific library:

`cs resolve {{group_id}}:{{artifact_id}}:{{artifact_version}} --what-depends-on {{searched_group_id}}:{{searched_artifact_id}}`
```

The same issue as above.
Optional fix: broken description: `specific` expected.

> - Print all the libraries that depends on a particoular library version:
> 
> `cs resolve {{groupId}}:{{artifactId}}:{{artifactVersion}} --what-depends-on {{searchedGroupId}}:{{searchedArtifactId}}{{searchedArtifactVersion}}`

Reviewer: ```suggestion
- Print all the libraries that depends on a specific library version:

`cs resolve {{group_id}}:{{artifact_id}}:{{artifact_version}} --what-depends-on {{searched_group_id}}:{{searched_artifact_id}}{{searched_artifact_version}}`
```

The same issue as above.

> - Print eventual conflits between a set of packages:
> 
> `cs resolve --conflicts {{groupId1}}:{{artifactId1}}:{{artifactartifactVersion1}} {{groupId2}}:{{artifactId2}}:{{artifactVersion2}}} {{groupId3}}:{{artifactId3}}:{{artifactVersion3}}`

Reviewer: ```suggestion
- Print eventual conflicts between a set of packages:

`cs resolve --conflicts {{group_id1:artifact_id1:artifact_version1 group_id2:artifact_id2:artifact_version2 ...}}`
```

Optional fix: broken description: `conflicts` expected.
Mandatory fix: [broken ellipsis](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases): ellipsis expected.
